public/style.css

```diff
@@ -0,0 +1,318 @@
+body {
+  background-color: #F5F5F5;
+}
+
+h1 {
+  font-family: Century Gothic;
+  text-align: center;
+   padding: 10px;
+}
+
+h2 {
+  font-family: Century Gothic;
+  text-align: center;
+   padding: 10px;
+}
+
+h3 {
+  font-family: Century Gothic;
+  text-align: center;
+   padding: 10px;
+}
+
+p {
+  font-family: Century Gothic;
+  text-align: center;
+}
+
+.login-container {
+  border: 1px solid grey;
+  height: 250px;
+  width: 350px;
+  /*padding: 0%;*/
+  margin: auto;
+  text-align: center;
+  background-color: #99CCCC;
+
+}
+/*navagation bar*/
+.bar {
+    list-style-type: none;
+    margin: 0;
+    padding: 0;
+    overflow: hidden;
+    background-color: #333;
+
+}
+
+.button {
+  margin-left: 50px;
+  margin-right: 10px;
+  display: block;
+  letter-spacing: 1.5px;
+  margin-top: 1.5em;
+  width: 10.3em;
+  height: 25px;
+  text-align: center;
+  background-color: #333;
+  color: white;
+  border: none;
+  outline-style: none;
+  font-family: Century Gothic;
+  text-decoration: none;
+}
+
+li {
+    float: left;
+}
+
+li a {
+    font-family: Century Gothic;
+    display: block;
+    color: white;
+    text-align: center;
+    padding: 14px 16px;
+    text-decoration: none;
+}
+/* Change the link color to #111 (black) on hover */
+li a:hover {
+    background-color: #111;
+}
+
+#username-login {
+    /*margin-top: 4em;*/
+    /*padding-top: 15px;*/
+    padding: 7px;
+    letter-spacing: 3.5px;
+    display: block;
+    text-align: center;
+    margin: 0 9.4em;
+}
+#password-login {
+    /*padding-top: 15px;*/
+    padding: 7px;
+    visibility: visible;
+    letter-spacing: 3.5px;
+    display: block;
+    text-align: center;
+    margin: 0 9.4em;
+}
+
+.login {
+  margin-top: 4em;
+}
+
+.submit {
+    margin-left: 50px;
+    display: block;
+    letter-spacing: 1.5px;
+    margin-top: 2.5em;
+    width: 23.3em;
+    height: 25px;
+    text-align: center;
+    background-color: #D5D5D5;
+    color: white;
+    /*border: none;*/
+    outline-style: none;
+}
+
+.vote-button {
+    margin-left: 50px;
+    display: block;
+    letter-spacing: 1.5px;
+    margin-top: 2.5em;
+    width: 12em;
+    height: 25px;
+    text-align: center;
+    margin: auto;
+    background-color: #D5D5D5;
+    color: white;
+    /*border: none;*/
+    outline-style: none;
+}
+
+.vote-button-down {
+    margin-left: 50px;
+    display: block;
+    letter-spacing: 1.5px;
+    margin-top: 2.5em;
+    width: 12em;
+    height: 25px;
+    text-align: center;
+    margin: auto;
+    background-color: #333;
+    color: #757575;
+    /*border: none;*/
+    outline-style: none;
+}
+
+.vote-button-down-pressed {
+    margin-left: 50px;
+    display: block;
+    letter-spacing: 1.5px;
+    margin-top: 2.5em;
+    width: 12em;
+    height: 25px;
+    text-align: center;
+    margin: auto;
+    background-color: #333;
+    color: pink;
+    /*border: none;*/
+    outline-style: none;
+    border: solid, 1px, yellow;
+}
```

Comment:
Similarly, all the styles that are common to all buttons should be on e.g. a `button` class. `.button-down-pressed` only really needs the few color differences. (N.B. an HTML tag can have multiple classes `<div class="button button-down-pressed">`)